pyproject.toml

```diff
@@ -27,6 +27,7 @@ include = [
 [tool.poetry.dependencies]
 python = "^3.7"
 napalm = {version = "^4.0.0", optional = true}
+pan-python = "^0.22.0"
```

Comment:
I don't see where this is used? 

Also, we would not support this as a required dependency.

Reply:
i will remove.

Reply:
I do not understand why the poetry lock file is changed if this is not needed?

Reply:
Sorry, I'm in the habit of running poetry update. I'll get this reverted.